boa_gc/src/internals/mod.rs

```diff
@@ -1,5 +1,4 @@
 mod ephemeron_box;
-pub(crate) use ephemeron_box::EphemeronBox;
-
 mod gc_box;
-pub(crate) use gc_box::GcBox;
+
+pub(crate) use {self::ephemeron_box::EphemeronBox, self::gc_box::GcBox};
```

Comment:
I think it would be better as `self::{...}`, scoping stuff inside of it, instead of repeating `self`